client/starwhale/core/dataset/tabular.py

```diff
@@ -215,18 +210,6 @@ def _transform(data: t.Any) -> t.Any:
                 return Binary(fp=data, auto_convert_to_bytes=True)
             elif isinstance(data, dict):
                 return {k: _transform(v) for k, v in data.items()}
-            elif isinstance(data, (list, tuple)):
```

Comment:
Only remove `Sequence`, keep `transform` for list and tuple?

Reply:
Fixed